.github/CONTRIBUTING.md

```diff
@@ -75,9 +75,8 @@ Follow the following steps to update documentations to their latest version:
 
 ## Coding conventions
 
-* two spaces; no tabs
-* no trailing whitespace; blank lines should have no spaces; new line at end-of-file
-* use the same coding style as the rest of the codebase
+Please use the same coding style as the rest of the codebase via [our `.editorconfig` file](../.editorconfig).
+Check out [EditorConfig.org](https://editorconfig.org/) to learn how to make your tools adhere to it.
```

Comment:
I'd like to keep the conventions written in prose, but referring to the EditorConfig is a good idea!

Reply:
I agree. It's pretty short so we can afford to have both the description and the info about EditorConfig.  Let's see if @katrinleinweber is still around, though.

Reply:
I am, but I already deleted my fork, thus can't edit this PR. Follow-up: https://github.com/freeCodeCamp/devdocs/pull/1445  ;-)